Update to 2.1.0 and add co-maintainer

synapticarbors opened the pull request:
<!--
Thank you for pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

synapticarbors committed:
Update to 2.1.0 and add co-maintainer

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

synapticarbors commented:
@conda-forge-admin, please rerender

Commit:
MNT: Re-rendered with conda-build 3.19.2, conda-smithy 3.7.0, and conda-forge-pinning 2020.04.28.13.02.26

synapticarbors committed:
fix licence file location

synapticarbors commented:
@conda-forge/pastedeploy -- Several automatic PRs updating the package have not been dealt with, so I've updated the recipe in this PR and added myself as a co-maintainer. Please have a look when you get the chance.

kfasolin commented:
Thanks, and welcome 👍
